### DISCUSSION TITLE
Cards don't show 99.9% of the time upon first visit on my own instance.

### DISCUSSION THREAD

**Discussion by bassamanator** [anuraghazra/github-readme-stats#3187] [Q&A] [answered]
I've deployed to Vercel, no changes made whatsoever. 

Upon first visit to my profile, the cards never show, _usually_ it works when I refresh. Third time's the charm.

Can I make any changes to my Vercel settings to make things work better?

Thanks!

**Comment by qwerty541** [comment, marked as answer]
@bassamanator This issue caused by timeout of GitHub CDN when it reuploads an image from your Vercel instance. In future it can be fixed by migration on GitHub actions. See for more info: https://github.com/anuraghazra/github-readme-stats/issues/2603

**Comment by bassamanator** [comment]
@qwerty541 Got it. Thank you!
